scripts/android-studio-build.gradle

```diff
@@ -0,0 +1,27 @@
+task buildAndroidStudioRelease(type: Zip) {
+    archiveName 'androidStudioRelease.zip'
+    destinationDir = buildDir
+    def pluginDirs = [":google-cloud-tools-plugin", ":google-account-plugin"]
```

Comment:
I'm noticing when I try out this script that the google-cloud-tools folder has the plugin sources.jar but not the actual code.jar.  google-account-plugin looks good though.

One other thing is that common-lib is missing the sources jar, and most of the lib jars don't have srcjars. Not sure if AS want that.